lib.php

```diff
@@ -7307,7 +7307,7 @@ function hsuforum_cm_info_view(cm_info $cm) {
     }
 
     if ($unread = hsuforum_count_forum_unread_posts($cm, $cm->get_course())) {
-        $out .= '<a class="unread" href="' . $cm->url . '">';
+        $out .= '<a class="unread" href="' . $cm->get_url() . '">';
```

Comment:
Same question as on 7296

Reply:
The URL attribute is a private attribute and the function returns the private value. I believe this is part of a PSR standard.